pkg/api/types/image_types.go

```diff
@@ -166,3 +166,11 @@ type ImageTagCommandOptions struct {
 	// Target is the image to be created.
 	Target string
 }
+
+type ImageRemoveCommandOptions struct {
```

Comment:
sorry to back and forth on this but we may need to change this to `ImageRemoveOptions` (naming is hard... 😅 ). Please see https://github.com/containerd/nerdctl/pull/1881#discussion_r1083312219 for a summary (so far), and https://github.com/containerd/nerdctl/issues/1889 for the discussion.

(also `processImageRemoveCommandOptions` -> `processImageRemoveOptions`)

Reply:
can you also add a brief command to avoid `export struct lack comment` warning? Something like `// ImageRemoveOptions specifies options for nerdctl rmi command.`

Reply:
Also from the discussion, we should move the `stdout io.Writer` to the option struct as well.

Reply:
I've read these discussions. I'll do this.